.github/scripts/generate-release-body.ts

```diff
@@ -213,10 +212,17 @@ async function main() {
   const emptyLabelPRs = prByLabels[''] || [];
```

Comment:
What about this part?

I mentioned it in the Slack - won't this be always empty if we have a CI check for labels?